test/integration/rule-content-route.yaml.tmpl

```diff
@@ -1,5 +1,5 @@
 destination: "{{.destination}}.{{.namespace}}.svc.cluster.local"
-precedence: 3
+precedence: 2
```

Comment:
testing PUT instead of POST. content and default route are named the same in the test driver, but the third rule uses precedence.